.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,103 @@
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Bugs
+description: Report a bug with the Microsoft build of Go
+title: "import/path: issue title"
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for helping us improve! 🙏 Please answer these questions and provide as much information as possible about your problem.
+
+  - type: input
+    id: go-version
+    attributes:
+      label: Microsoft build of Go version
+      description: |
+        What version of Go are you using (`go version`)?
```

Comment:
We might want to ask for more info here, since go version doesn't give the revision. Until we get:

* https://github.com/microsoft/go/issues/262